From fa9e3d0c6ca23ab5b00d112be39bbed6a1749b66 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@st.com>
Date: Fri, 22 Sep 2023 13:56:17 +0200
Subject: [PATCH 349/841] feat(stm32mp2-fdts): update STM32MP257F-EV1 DT

Add include for DDR configuration, and reference to OTP storing the
board ID.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 602ab995d4d3d1c44dba8c86158e7458781ed1cf feat(stm32mp2-fdts): update STM32MP257F-EV1 DT]
Signed-off-by: Yann Gautier <yann.gautier@st.com>
Change-Id: Ie2d5272ecf1dac77b91b2c148ec4dc1fb7b76631
---
 fdts/stm32mp257f-ev1.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index 62ed0bea0..17363f28c 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -10,6 +10,7 @@
 #include "stm32mp257.dtsi"
 #include "stm32mp25xf.dtsi"
 #include "stm32mp257f-ev1-ca35tdcid-rcc.dtsi"
+#include "stm32mp25-ddr4-2x16Gbits-2x16bits-1200MHz.dtsi"
 #include "stm32mp25-pinctrl.dtsi"
 #include "stm32mp25xxai-pinctrl.dtsi"
 
@@ -40,6 +41,12 @@
 	};
 };
 
+&bsec {
+	board_id: board_id@3d8 {
+		reg = <0x3d8 0x4>;
+	};
+};
+
 &hash {
 	status = "okay";
 };
-- 
2.47.2

